x-pack/platform/plugins/shared/saved_objects_tagging/public/ui_api/index.ts

```diff
@@ -51,7 +51,7 @@ export const getUiApi = ({
     convertNameToReference: buildConvertNameToReference({ cache }),
     getTagIdsFromReferences,
     getTagIdFromName: (tagName: string) => convertTagNameToId(tagName, cache.getState()),
-    updateTagsReferences,
+    updateTagsReferences: replaceTagReferences,
```

Comment:
Previously, the `utils` module re-exported the `replaceTagReferences` function as `updateTagsReferences`. Now we're directly importing `replaceTagReferences` from `common`.